src/main/frontend/extensions/hickory.cljc

```diff
@@ -0,0 +1,12 @@
+(ns frontend.extensions.hickory
```

Comment:
To bypass the "hictory doesnt work in Node" mentioned in https://github.com/logseq/logseq/blob/17966ed5654349c0c0894771c28b857f7f9eeb8e/src/test/frontend/extensions/zotero/extractor_test.cljs#L1-L3